src/plugins/intel_gpu/src/graph/graph_optimizer/pre_replace_deconv.cpp

```diff
@@ -277,7 +277,9 @@ void pre_replace_deconv::run(program& p) {
                 p.add_connection(input_node, conv_node);
 
                 {
-                    auto weights_node_conv_rpl_ptr = p.nodes_map.find(weight_replace_node_id)->second;
+                    auto iter = p.nodes_map.find(weight_replace_node_id);
+                    OPENVINO_ASSERT(iter != p.nodes_map.end(), "[Coverity] ", weight_replace_node_id, " is not in nodes_map.");
```

Comment:
[optional] Is "[Coverity]" needed?

Reply:
It just shows why it prints the assert messages.